api/go.mod

```
@@ -61,9 +61,9 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.29.0 // indirect
+	k8s.io/apiextensions-apiserver v0.29.2 // indirect
```

```suggestion
	k8s.io/apiextensions-apiserver v0.29.3 // indirect
```
Same for all 0.29.2 versions.

---

Thanks. It's done now.